<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimplePTTCommandOutputDialog</class>
 <widget class="QDialog" name="SimplePTTCommandOutputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>547</width>
    <height>380</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Simple PTT command output</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/sdrangel_icon.png</normaloff>:/sdrangel_icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="processLayout">
     <item>
      <widget class="QLabel" name="statusIndicator">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Idle</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel { background-color: gray; border-radius: 12px; }</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="endTime">
       <property name="minimumSize">
        <size>
         <width>146</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Liberation Sans</family>
         <pointsize>8</pointsize>
        </font>
       </property>
       <property name="toolTip">
        <string>End time of last execution</string>
       </property>
       <property name="text">
        <string>2015-01-01 00:00:00.000</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="errorExitLayout">
     <item>
      <widget class="QLabel" name="errorLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Error:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errorText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Process error status</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exitLabel">
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Exit:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exitCode">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Return code</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="exitText">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Process exit status</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="logEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="font">
      <font>
       <family>Monospace</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="toolTip">
      <string>Output log (stdout + stderr)</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SimplePTTCommandOutputDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SimplePTTCommandOutputDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
